src/sass/app_2/_common/loading.scss

```diff
@@ -51,15 +70,57 @@
     }
 }
 
-@-webkit-keyframes sk-stretchdelay {
-    /* postcss-bem-linter: ignore */
-    0%, 40%, 100% {
-        -webkit-transform: scaleY(1);
-        transform: scaleY(1);
+/** @define initial-loader */
+.initial-loader {
+    width: 100vw;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    display: flex;
+    @include themify($themes) {
+        background-color: themed('background_cover');
     }
-    /* postcss-bem-linter: ignore */
-    20% {
-        -webkit-transform: scaleY(2);
-        transform: scaleY(2);
+
+    &__wrapper {
+        @include toEm(margin-top, 56px, 12px);
+    }
+    &__circ {
+        margin: 0.8em;
+        border-radius: 20px;
+        height: 100%;
+        width: 0.6em;
+        display: inline-block;
+
+        &--1 {
+            width: 24px;
+            height: 24px;
+            background-color: $COLOR_WHITE;
```

Comment:
can be a mixin?

Reply:
Still don't see the mixin 😄`&--1, &--2, &--3, &--4` are all the same except for the animation.

alternatively you could add the properties that are the same with `&--1, &--2, &--3, &--4 {...}`

Reply:
i deleted the class since we don't use it anymore 🙏